rest-backend/src/main/java/org/eclipse/steady/backend/model/Library.java

```diff
@@ -461,7 +461,7 @@ public Set<ProgrammingLanguage> getDevelopedIn() {
    * @return a {@link org.eclipse.steady.backend.model.ConstructIdFilter} object.
    */
   @JsonProperty(value = "constructTypeCounters")
-  @JsonView(Views.Default.class)
+  @JsonView(Views.CountDetails.class)
```

Comment:
This change would fix the mitigation tab but would probably break the "bloat" tab. I would suggest to adopt the fix suggested in issue #573

Reply:
Tried that, but it does not fix the mitigation tab. The fix above, however, makes that the column `Library Executable Constructs` on the bloat tab remains empty (I guess that is the problem you anticipated).